### DISCUSSION TITLE
SplitPartitioning

### DISCUSSION THREAD

**Discussion by rpenaran** [microsoft/msquic#2391] [Q&A] [answered]
Hi! As far as I understand, a given connection is allocated into a worker which is running in a thread. So, the send/receive logic is in the same thread. But, when going deeper into the code, I saw a exec profile that uses SplitPartitioning. What is that feature doing? Is it splitting the send and receive logic into two different threads?

**Comment by nibanks** [comment, marked as answer]
SplitPartitioning isn't fully fleshed out yet. The idea there is that you split partitions to match hyper-threaded cores; with UDP for a connection on one and QUIC layer on another.

**Reply by rpenaran** [reply]
I see. Thanks!
